mofacts/client/views/experiment/card.js

```diff
@@ -1973,11 +1973,18 @@ function beginQuestionAndInitiateUserInput(delayMs, deliveryParams) {
       });
     }, timeuntilaudio);
   } else { // Not a sound - can unlock now for data entry now
-    const questionToSpeak = currentDisplay.text || currentDisplay.clozeText;
+    const questionToSpeak = currentDisplay.clozeText || currentDisplay.text;
```

Comment:
Why did you switch these?

Reply:
In the testStim.json file the question is stored in clozeText instead of text. So the API was reading text while clozeText was appearing on the screen.